File: Data Structures & Algorithms/valid-sudoku/submission-0.cpp
class Solution {
public:
    bool isValidSudoku(vector<vector<char>>& board) {
          vector<unordered_set<char>> rows(9);
  vector<unordered_set<char>> cols(9);
  vector<unordered_set<char>> squares(9);

  for (int i = 0; i < 9; i++) {
    for (int j = 0; j < 9; j++) {
      char num = board[i][j];
      if (num == '.') {
        continue;
      }
      int squareIndex = (i / 3) * 3 + (j / 3);
      if (rows[i].count(num) || cols[j].count(num) ||
          squares[squareIndex].count(num)) {
        return false;
      }
      rows[i].insert(num);
      cols[j].insert(num);
      squares[squareIndex].insert(num);
    }
  }
  return true;

    }
};

File: Data Structures & Algorithms/valid-palindrome-ii/submission-0.py
class Solution:
    def validPalindrome(self, s: str) -> bool:
        i = 0
        j = len(s) - 1
        count = 0
        while i < j:
            if s[i] != j[i]:
                count +=1
            i += 1
            j -= 1
        return True if count <= 0 else False